about.html

```diff
@@ -1,64 +1,114 @@
-<!doctype html>
+<!DOCTYPE html>
 
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-
-  <title>Sprint Challenge - About</title>
-
-  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
-
-</head>
-
-<body>
+  <head>
+    <meta charset="utf-8" />
+
+    <title>Sprint Challenge - About</title>
+
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto|Rubik"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/index.css" />
+  </head>
+
+  <body>
+    <header>
+      <nav class="top-nav">
+        <img
+          class="logo"
+          src="C:\Users\snadi\Sprint-UI-CSS\web-Sprint-Challenge-User-Interface-and-Git\img\lambda-black.png"
+          alt="Lambda Logo"
+        />
+        <a href="C:\Users\snadi\Sprint-UI-CSS\web-Sprint-Challenge-User-Interface-and-Git\index.html"
```

Comment:
You almost got it! You are referencing items in your file structure incorrectly (I'm talking about the hrefs/src lines 21,28,38 etc...). 

Here are some tips to reference properly:

• A leading slash tells the browser to start at the root directory.
• If you don't have the leading slash, you're referencing from the current directory.
• If you add two dots before the leading slash, it means you're referencing the parent of the current directory.

For example, in our index.js that is located in our root directory, we want to reference about.html which is also located in our root directory. You can do something like this:

```
<a href"/home.html"> Home </a>
<a href"/about.html"> About </a>
```